modules/dnn/src/webgpu/include/buffer.hpp

```diff
@@ -12,10 +12,10 @@ class Buffer
 // #ifdef HAVE_WEBGPU
 public:
     Buffer(const std::shared_ptr<wgpu::Device> device);
-    Buffer( const std::shared_ptr<wgpu::Device> device, 
+    Buffer(const std::shared_ptr<wgpu::Device> device, 
             const void* data, size_t size, 
             wgpu::BufferUsage usage = wgpu::BufferUsage::Storage | 
-            wgpu::BufferUsage::CopySrc | wgpu::BufferUsage::CopySrc );
+            wgpu::BufferUsage::CopyDst | wgpu::BufferUsage::CopySrc );
```

Comment:
Unneccesary blank in "wgpu::BufferUsage::CopySrc );", incorrect indent for the parameters list.

Please check coding style conformance for all source files. 
https://github.com/opencv/opencv/wiki/Coding_Style_Guide

Reply:
Thanks, I've fix the coding style issue.